addons/website_sale_delivery/migrations/10.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+COLUMN_COPIES = {
+    'delivery_carrier': [
+        ('website_description', None, None),
+    ],
+}
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.copy_columns(env.cr, COLUMN_COPIES)
```

Comment:
This line gives error on Travis. Are you sure the content is not already on HTML?

Reply:
No, It's a related field to product.description_sale.

The conversion to html it must be done in product migration.